_posts/2018-01-10-meeting-notes.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "The New HV Open Website"
+date: 2018-03-26 15:00:00 -0400
```

Comment:
probably update this to current time, also, it would be worth while to have a more descriptive url than meeting-notes, because of how the url is constructed